frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -720,19 +729,33 @@ export const TreeViewer = ({ data, heading }) => {
           />,
         ]}
         // Search bar for the next phase
-        // right={
-        //   <TextField
-        //     label="Search"
-        //     variant="outlined"
-        //     size="small"
-        //     classes={{
-        //       root: styles.search,
-        //     }}
-        //   />
-        // }
+        right={[
```

Comment:
I thought it added a good UI element, and would allow us to keep it in consideration when developing. Right now its just UI element, and does nothing, but i dont think that's a bad thing bc the usability testing would not involve any testing tasks anyways ukwim?

Reply:
I suppose, I think that it could cause confusion though for people if there is an element visible which is not actually working, but I don't hate the idea of leaving it if you'd like

Reply:
Ya I think it adds a nice "full-ness" to the UI ukwim? Do you think we could mitigate this by making it "fake work"? Like no matter what you search, it highlights the same node, or something like that?

Reply:
I guess we could....bring it up during meeting?

Reply:
Sure! Should we merge it for now, especially since removal is quite easy?

Reply:
sure

Reply:
sure lets leave it for now